O/OpenCL/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OpenCL"
+version = v"2022.9.23"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/KhronosGroup/OpenCL-Headers.git", "4c50fabe3774bad4bdda9c1ca92c82574109a74a"),
```

Comment:
This should depend on `OpenCL_Headers` as a [`BuildDependency`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.BuildDependency) if they're needed for building this package.  Headers are useless at runtime and this would create a useless clash with the other package.
```suggestion
```

Reply:
But OpenCL_Headers was built with a different version. Wouldn't it be a good idea to bundle headers + library into a single place to avoid API issues in the future?

Reply:
You can update `OpenCL_Headers_jll` then.  Headers aren't needed at runtime, not sure what API issues you're referring to.  If you need a particular version for building this package, you can specify the compat with `OpenCL_Headers_jll`.